### ISSUE TITLE
Conch tests fail on Python 3 test_manhole_tap ,    twisted.conch.test.test_mixin, twisted.conch.test.test_scripts,       twisted.conch.test.test_unix

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8691]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8691|
|Type|enhancement|
|Created|2016-07-26 14:35:15Z|
|Branch|https://github.com/twisted/twisted/tree/8691-rodrigc-conch|

These conch tests fail on Python 3:

test_manhole_tap
test_mixin
test_scripts
test_unix

<details><summary>Searchable metadata</summary>

```
trac-id__8691 8691
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8691_rodrigc_conch 8691-rodrigc-conch
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469543715480428 1469543715480428
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/421

**Comment by twisted-trac** [comment]
|[<img alt="acabhishek942's avatar" src="https://avatars.githubusercontent.com/u/7778186?s=50" width="50" height="50">](https://github.com/acabhishek942)<a name="note_2"></a>|@acabhishek942 commented|
|-|-|

looks good to me. Please merge

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 1aac4c147368dcb5cbd1967ccf8bca0c41a45be0
```
#!CommitTicketReference repository="" revision="1aac4c147368dcb5cbd1967ccf8bca0c41a45be0"
Merge pull request #421 from twisted/8691-rodrigc-conch

Author: rodrigc
Reviewer: acabhishek942, einarfd
Fixes: #8691

Python 3 fixes for conch
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
